test/efficiency_tests/test_linalg_time.py

```diff
@@ -17,3 +18,16 @@ def test_inv_sym(hook, workers):
     x = torch.randn(N, K).fix_precision().share(bob, alice, crypto_provider=crypto_prov)
     gram = x.t().matmul(x)
     gram_inv = inv_sym(gram)
+
+
+@assert_time(max_time=20)
+def test_norm_mpc(hook, workers):
+    torch.manual_seed(42)  # Truncation might not always work so we set the random seed
+    bob = workers["bob"]
+    alice = workers["alice"]
+    crypto_prov = sy.VirtualWorker(hook, id="crypto_prov")
+
+    n = 100
+    t = torch.randn([n])
+    t_sh = t.fix_precision(precision_fractional=6).share(bob, alice, crypto_provider=crypto_prov)
+    norm_sh = _norm_mpc(t_sh, norm_factor=n ** (1 / 2))
```

Comment:
Add an assertion to check the norm returned is not to bad (ie ||result - expected|| < e-2 par exemple))

Reply:
oh actually you did it, this test is a duplicate of the one in test/torch/linalg/test_operations.py
where the `@assert_time(max_time=20)` is missing however

Reply:
Yes... actually one test is located in the `efficiency_tests` repo and the other in `linalg/test_operations.py`. I agree this is repeated code, but as we have created a repo only for efficiency tests I was kind of obligated to do like that... But to be honest I did not like it very much.

Do you think I should remove the repeated test from `efficiency_tests` and just include the decorator in `linalg/test_operations.py` ?

Reply:
Ah ok I wasn't aware of efficiency_tests 
Not sure why we created this instead of just putting the @assert_time(max_time=20)  where needed 
But let's keep it like this for the moment I suppose

Reply:
It was an idea from @mari-linhares and me that was added to the tests 2 weeks ago.

Now, I think we will have a lot of repeated code (and longer tests) if we keep doing like that. In my opinion, it's better to just put the `assert_time` decorators on the tests in their respective repos when needed.

If you agree I will start doing it in this PR.

Reply:
Excellent let's do this!
I think this initial idea was a good one, but that we should put there only test of demos where we want to benchmark the efficiency, not specific tests. It would give us a way to monitor how our changes affect out flagship usecases